homeassistant/components/onboarding/views.py

```diff
@@ -98,8 +99,17 @@ class UserOnboardingView(_BaseOnboardingView):
                 await hass.components.person.async_create_person(
                     data['name'], user_id=user.id
                 )
+
             await self._async_mark_done(hass)
 
+            # Return an authorization code to allow fetching tokens.
+            auth_code = hass.components.auth.create_auth_code(
```

Comment:
Do we have expiring time for `auth_code`? 5 minutes?

Reply:
10 minutes

https://github.com/home-assistant/home-assistant/blob/f705ac6b43af601af41c32fb11b6da0859ed70cb/homeassistant/components/auth/__init__.py#L428-L433

Reply:
So for now, it might be okay. But in future, if on-boarding takes longer than 10 minutes, the auth_code will be expired

Reply:
If we get more steps in onboarding, I actually want to establish a websocket connection so we can use the APIs as usual. I guess in that case, we will ask users if they want to stay logged in during onboarding.